components/playlist/browser/resources/components/header.tsx

```diff
@@ -64,14 +69,36 @@ export default function Header ({ playlistId }: HeaderProps) {
           <StyledLink to='/'>
             <ColoredIcon name='arrow-left' color={color.icon.default} />
           </StyledLink>
-          <PlaylistInfo
+          <StyledPlaylistInfo
             isDefaultPlaylist={playlist.id === 'default'}
             itemCount={playlist.items.length}
             playlistName={playlist.name}
             totalDuration={0}
             nameColor={color.text.primary}
             detailColor={color.text.secondary}
           />
+          <ContextualMenuAnchorButton
+            items={[
+              { name: 'Rename', iconName: 'edit-box', onClick: () => {} },
+              {
+                name: 'Edit',
+                iconName: 'list-bullet-default',
+                onClick: () => {}
+              },
+              { name: 'Share', iconName: 'share', onClick: () => {} },
```

Comment:
I'm curious if this 'share' icon could be interpreted as 'share-macos' and 'share-windows' depending on the platform. Actually, our design guide says this icon should be share-macos.

Reply:
It won't changed based on platform, nah - do you reckon it'd be a useful thing to add to Leo?

Reply:
I'm not sure, but depending on what @aguscruiz was intended. But now I think I should rever this back to 'share-macos' 😄

@aguscruiz , when you use 'share-macos' icon, do you mean to use 'share-macos' across all platform, or use platform specific icon?
If it's the latter, I would automatically interpreted Leo icon would be nice. It's interesting that we don't have 'share-linux' icon though.